app/gui/view/graph-editor/src/component/node/action_bar.rs

```diff
@@ -31,9 +33,11 @@ const BUTTON_OFFSET: f32 = 0.5;
 /// Grow the hover area in x direction by this amount. Used to close the gap between action
 /// icons and node.
 const HOVER_EXTENSION_X: f32 = 15.0;
+const VISIBILITY_TOOLTIP_LABEL: &str = "Show preview";
```

Comment:
It seems the old labels were not sentences. But if these are, should they be followed by a period?